### PR TITLE
DevEx: use docketNumber instead of caseId for UserCase

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/presenter/actions/FileDocument/submitRespondentCaseAssociationRequestAction.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const submitRespondentCaseAssociationRequestAction = async ({
   applicationContext,
   get,
 }) => {
-  const { caseId } = get(state.caseDetail);
+  const docketNumber = get(state.caseDetail.docketNumber);
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/FileDocument/submitRespondentCaseAssociationRequestAction.js:15]: 🎉

**Review by rachelschneiderman — APPROVED** [review]
Does the swagger.json need to be updated?

File: shared/src/persistence/dynamo/cases/getCasesByUser.js [hunk 2]
```diff
@@ -18,6 +18,8 @@ exports.getCasesByUser = async ({ applicationContext, userId }) => {
     })
   ).map(mapping => mapping.sk.split('|')[1]);
 
+  console.log('caseIds here arererere', JSON.stringify(caseIds));
```
Comment (ericsorenson) [inline shared/src/persistence/dynamo/cases/getCasesByUser.js]: 🙃

**Review by ericsorenson — APPROVED** [review]
Just one console.log!

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**